### PR TITLE
Upgrade scripts into a working ETL

### PR DESCRIPTION
**Pull request by jobar**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 docopt
-hdfs
\ No newline at end of file
+hdfs
+impyla
```
Comment (halfak) [inline requirements.txt:3]: I don't see where `impyla` is being used anywhere.

File: python/wiki_etl.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    fulltext_table = "{0}_fulltext".format(metadata_table)
+
+    wiki_xml = wiki_path(FOLDER_XMLBZ2)
+    wiki_json = wiki_path(FOLDER_JSONBZ2)
+    wiki_meta = wiki_path(FOLDER_METADATA)
+
+    logger.debug("Launching download_dump")
+    download_dump_args = copy.copy(args)
+    download_dump_args["<hdfs-path>"] = wiki_xml
+    download_dump.main(download_dump_args)
```
Comment (halfak) [inline python/wiki_etl.py]: Nice

**Comment by halfak** [comment]
Just a couple of notes.  Otherwise looks good.  Did you run the whole ETL in a test?

File: requirements.txt (hunk 1 above)
Comment (jobar) [inline requirements.txt:3]: hql_runner - Package is named impyla, but import is impala

File: python/wiki_etl.py (hunk 2 above)
Comment (jobar) [inline python/wiki_etl.py]: @halfak : Is the hack line 120 ok for you (even if ugly)?

**Comment by jobar** [comment]
Closing, everything was merged in #7

**Pull request closed without merging**